packages/components/src/internal/renderers/MultiValueRenderer.tsx

```diff
@@ -28,37 +28,53 @@ export const MultiValueRenderer: FC<MultiValueRendererProps> = memo(({ data }) =
     let i = -1;
     return (
         <div>
-            {data.map((item, key) => {
-                let text: ReactNode;
-                let url: string;
+            {data
+                .map((item, key) => {
+                    let text: ReactNode;
+                    let url: string;
 
-                if (Map.isMap(item)) {
-                    if (item.has('formattedValue')) {
-                        text = item.get('formattedValue');
+                    if (Map.isMap(item)) {
+                        if (item.has('formattedValue')) {
+                            text = item.get('formattedValue');
+                        } else {
+                            const o = item.has('displayValue') ? item.get('displayValue') : item.get('value');
+                            text = o !== null && o !== undefined ? o.toString() : null;
+                        }
+
+                        url = item.get('url');
+                    } else if (item !== undefined && item !== null) {
+                        text = item.toString();
                     } else {
-                        const o = item.has('displayValue') ? item.get('displayValue') : item.get('value');
-                        text = o !== null && o !== undefined ? o.toString() : null;
+                        return null;
                     }
 
-                    url = item.get('url');
-                } else if (item !== undefined && item !== null) {
-                    text = item.toString();
-                } else {
-                    return null;
-                }
+                    if (text === undefined || text === null || text === '') return null;
 
-                if (text === undefined || text === null || text === '') return null;
+                    // If the string has \n characters, replace them with <br/> elements
+                    if (typeof text === 'string' && text.indexOf('\n') > -1) {
```

Comment:
This could also be accomplished with CSS `white-space: pre-line`:

```tsx
text = (
  <span style={{ whiteSpace: 'pre-line' }}>
    {text}
  </span>
);
```